src/browser/starter.js

```diff
@@ -1214,7 +1209,7 @@ V86Starter.prototype.create_file = function(file, data, callback)
  * @param {function(Object, Uint8Array)} callback
  * @export
  */
-V86Starter.prototype.read_file = function(file, callback)
+V86Starter.prototype.read_file = async function(file, callback)
```

Comment:
(same as above). We should probably remove the callback and return a promise instead.